package.json

```diff
@@ -143,5 +143,8 @@
   },
   "dependencies": {
     "react-dev-utils": "^12.0.1"
+  },
+  "volta": {
```

Comment:
Can we please remove this section?

Reply:
yes we can remove it, but i think we need to mention the node.engine so others know what node version they need to use.

Reply:
it's removed, and i add these lines:
```json
"engines": {
    "node": "~18.20.3"
  },
```